features/step_definitions/logging.rb

```diff
@@ -34,12 +35,12 @@
   # Create namespace
   unless project('openshift-logging').exists?
     # oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/namespace.yaml
-    namespace_yaml = "https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/openshift/ci-operator/build-image/openshift-logging-namespace.yaml"
+    namespace_yaml = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/deploy_clo_via_olm/4.2/01_clo_ns.yaml"
     @result = admin.cli_exec(:create, f: namespace_yaml)
     raise "Error creating namespace" unless @result[:success]
   end
   unless project('openshift-operators-redhat').exists?
-    eso_namespace_yaml = "https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/openshift/ci-operator/build-image/openshift-operators-redhat-namespace.yaml"
+    eso_namespace_yaml = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/eleasticsearch/deploy_via_olm/4.2/01_eo-project.yaml"
```

Comment:
@QiaolingTang  @pruan-rht  as 4.1 and 4.2 have different resource files. I am thinking how to use a common step to deploy the operators.
@pruan-rht  Could we get the clusterversion in cucushift?

Reply:
@anpingli to get the clusterverison, the user has to be a cluster-admin

Reply:
@pruan-rht  Is there a global variable to return clusterversion.

Reply:
Dev is aware of the issue and promised to fix it so a normal user can get that informaiton. 
 Currently, if the user is a clusteradmin, then make the call `cluster_version('version').verison` would do return that information.

Reply:
Since the installation require `admin` privilege, we can use the call above to decide which yaml to use

Reply:
Thanks Anping and Peter, I updated my PR.